git_machete/github.py

```diff
@@ -100,7 +101,20 @@ def get_token_from_hub() -> Optional[str]:
     def get_token_from_env() -> Optional[str]:
         return os.environ.get(GITHUB_TOKEN_ENV_VAR)
 
-    return get_token_from_env() or get_token_from_gh() or get_token_from_hub()
+    def get_token_from_file_in_root() -> Optional[str]:
+        required_file_name = '.github-token'
+        file_full_path = f'{ROOT_DIR}/{required_file_name}'
```

Comment:
Whooops isn't `ROOT_DIR` just the installation directory of `git-machete` on the given machine (rather than root directory of the current project)? I'd rather expect `git_machete.git_operations.GitContext.get_root_dir` for this purpose